### PR TITLE
feat(chains): adds scroll

### PR DESCRIPTION
**Pull request by 0xged**

### CODE REVIEW

**Review by 0xsambugs — COMMENTED** [review]
Looks good! But in order for the aggregator to support the chain, we should also:
1. Make sure the Multicall service supports scroll. Check `SUPPORTED_CHAINS`
2. Add support for price sources, such as Coingecko and DefiLlama

File: src/services/multicall/multicall-service.ts [hunk 1]
```diff
@@ -170,9 +170,15 @@ export class MulticallService implements IMulticallService {
 }
 
 const SUPPORTED_CHAINS: ChainId[] = [
-  1, 3, 4, 5, 10, 14, 16, 18, 19, 25, 30, 31, 40, 42, 56, 66, 69, 97, 100, 106, 108, 114, 122, 128, 137, 250, 288, 321, 420, 592, 1088, 1284,
-  1285, 1287, 2001, 4002, 8217, 8453, 9000, 9001, 42161, 42170, 42220, 42262, 43113, 43114, 44787, 59144, 71401, 71402, 80001, 84531, 421611,
-  421613, 11155111, 1313161554, 1666600000,
+  1, 3, 4, 5, 10, 14, 16, 18, 19, 25, 30, 31, 40, 42, 42, 44, 46, 50, 51, 56, 57, 61, 66, 69, 82, 83, 97, 100, 106, 108, 109, 114, 122, 128, 137,
```
Comment (0xsambugs) [inline src/services/multicall/multicall-service.ts:173]: What are these? You added a lot of chains, where did you get these ids from?

**Review by 0xsambugs — APPROVED** [review]
Looks good! Just one question for you

File: src/services/multicall/multicall-service.ts (hunk 1 above)
Comment (0xged) [reply src/services/multicall/multicall-service.ts:173]: All from here: https://github.com/mds1/multicall/blob/main/deployments.json

**Pull request merged**